wagtail/project_template/home/templates/home/welcome_page.html

```diff
@@ -0,0 +1,377 @@
+                </g>
+              </g>
+            </g>
+            <path fill="#9FD6E4" d="M75.325 68.613h.002-.002"/>
+            <path fill="#C7462F" d="M95.425 69.054h-.174.175c15.502 0 30.138-.877 43.064-2.432 2.372-5.235 3.81-10.982 4.097-17.027-.288 6.045-1.725 11.792-4.098 17.027-12.927 1.555-27.563 2.432-43.065 2.432m-.23 0h-.114.114m-.198 0h-.085.086m-.18 0h-.07.07m-.192 0h-.034.034m-.178 0h-.008.008"/>
+            <path fill="#1D2533" d="M137.33 26.445l-.09-.18.09.18m-.095-.19l-.036-.072c.01.023.024.047.035.072m-.06-.12l-.015-.03.016.03"/>
+            <path fill="#FFFFFE" d="M31.976 65.48c0 1.978-3.635 3.58-8.118 3.58-4.484 0-8.118-1.602-8.118-3.58 0-1.977 3.634-3.58 8.118-3.58 4.483 0 8.118 1.603 8.118 3.58M170.26 65.48c0 1.978-3.634 3.58-8.118 3.58s-8.118-1.602-8.118-3.58c0-1.977 3.634-3.58 8.118-3.58s8.118 1.603 8.118 3.58M58.957 76.666c0 1.977-3.634 3.58-8.117 3.58-4.483 0-8.118-1.603-8.118-3.58 0-1.976 3.635-3.58 8.118-3.58 4.483 0 8.117 1.604 8.117 3.58M145.525 76.666c0 1.977-3.634 3.58-8.117 3.58-4.484 0-8.118-1.603-8.118-3.58 0-1.976 3.634-3.58 8.118-3.58 4.483 0 8.117 1.604 8.117 3.58M103.93 80.022c0 1.977-3.636 3.58-8.12 3.58-4.483 0-8.118-1.603-8.118-3.58 0-1.976 3.635-3.58 8.12-3.58 4.482 0 8.117 1.604 8.117 3.58"/>
+          </g>
+        </g>
+      </g>
+    </svg>
+  </div>
+  <br>
+  <h2>{% templatetag openblock %} trans "Welcome to your new Wagtail site!" {% templatetag closeblock %}</h2>
+  <p>{% templatetag openblock %} trans 'Please feel free to <a href="https://github.com/wagtail/wagtail/wiki/Slack" target="_blank">join our community on Slack</a>, or get started with one of the links below.' {% templatetag closeblock %}</p>
+</div>
+<footer class="u-clearfix">
+  <a href="https://docs.wagtail.io/" target="_blank" rel="nofollower noopener">
+    <div class="option one">
+      <svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+        <defs>
+          <path d="M0 0h24v24H0V0z" id="a"></path>
+        </defs>
+        <clipPath id="b">
+          <use overflow="visible" xlink:href="#a"></use>
+        </clipPath>
+        <path clip-path="url(#b)" d="M9 21c0 .55.45 1 1 1h4c.55 0 1-.45 1-1v-1H9v1zm3-19C8.14 2 5 5.14 5 9c0 2.38 1.19 4.47 3 5.74V17c0 .55.45 1 1 1h6c.55 0 1-.45 1-1v-2.26c1.81-1.27 3-3.36 3-5.74 0-3.86-3.14-7-7-7zm2.85 11.1l-.85.6V16h-4v-2.3l-.85-.6C7.8 12.16 7 10.63 7 9c0-2.76 2.24-5 5-5s5 2.24 5 5c0 1.63-.8 3.16-2.15 4.1z"></path>
+      </svg>
+      <div>
+        <h4>{% templatetag openblock %} trans "Wagtail Documentation" {% templatetag closeblock %}</h4>
+        <p>{% templatetag openblock %} trans "Topics, references, &amp; how-to's" {% templatetag closeblock %}</p>
+      </div>
+    </div>
+  </a>
+  <a href="https://docs.wagtail.io/en/latest/getting_started/tutorial.html" target="_blank" rel="nofollower noopener">
```

Comment:
For this and the release notes link – would it be possible to point to the docs for the version of Wagtail that was used?

Reply:
I added this in a way that will work unless you're using a prerelease version of Wagtail (I only noticed because I happened to still be running 2.2a0 in my local environment). If anyone has any ideas on how to handle it when `{% wagtail_version %}` returns a prerelease version, let me know.